website/src/data/users.tsx

```diff
@@ -229,7 +229,16 @@ const Users: User[] = [
   },
 
   //-------  ALL OTHER TEMPLATES go here, can be searched for by name
-
+  {
+    title: 'Python (Django) Web App with PostGreSQL',
+    description:
+      'A restaurants review web app that uses whitenoise for static assets and PostGreSQL for the database. Ready for deployment on App Service with a PostGreSQL Flexible Server inside a VNet.',
+      preview: require('./images/msdocs-django-postgresql-sample-app-azd.png'),
+    website: 'https://github.com/pamelafox',
+    author: 'Pamela Fox',
+    source: 'https://github.com/pamelafox/msdocs-django-postgresql-sample-app-azd',
```

Comment:
I think @pamelafox was going to move this to azure-samples?

Reply:
Yes, still waiting for ghrnotify@ to unlock the forked repository (they automatically lock all forks of repos within the same org, as it's often a mistake, but I believe ours is a legitimate use).

Reply:
The content team decided they're comfortable with the changes and would rather not have forks lying around, so the changes are now in the official Azure-samples repo:
https://github.com/Azure-Samples/msdocs-django-postgresql-sample-app

Not sure what the author should be in this case? I authored the azd changes but it's now hosted in Azure-samples and the original author is the content team.

Reply:
"Azure Content Team" as author is fine.

Reply:
Sounds good, these changes should be ready to make now, by the way.